CuratedContent/EthicalOpenSource.md

```diff
@@ -0,0 +1,62 @@
+## Ethical Open Source Licensing
+
+<!--- deck text start --->
+Is your open source code being used for good or evil?
+<!--- deck text end --->
+
+#### Contributed by [Mark C. Miller](https://github.com/markcmiller86)
+#### Publication date: February, 15, 2025
+
+Resource information | Details
+:--- | :---
+title | 
+Authors |
+Website |
+
+In the last 20 years, the meaning of *open* in "open-source" has evolved significantly.
+Originating out of principles of freedom, transparency, and collaboration, open-source licenses govern the use, modification, and distribution of software and, increasingly, now data. 
+
+The open-source movement traces its origins to Richard Stallman’s creation of the **GNU General Public License (GPL)** in 1989.
+A restriction of GPL is that whenever GPL software is used in other software, that other software must itself also be openly and freely available under the terms of GPL.
+This *viral* design sought to supercede proprietary interests and ensure that software remains a shared resource free to be modified and redistributed.
+Legally speaking, GPL prevents commercial companies developing proprietary software from using any GPL software in their products. 
+
+Eventually, commercial interests subverted this design and permissive open source licenses took hold.
```

Comment:
I don't believe it was just commercial interests that didn't like the GPL and it's viral aspect.  I recall that many open source advocates disliked it as well.  The more permissive licenses: BSD, MIT, etc. I believe might actually pre-date the GPL and as such were not reactions to the GPL (as is kind of implied here).

Reply:
I stand corrected and did not know this tidbit prior to reading material about licensing. Thank you. Content substantially revised making the above now no longer applicable.